grafana/dashboards/cmode/harvest_dashboard_cluster.json

```diff
@@ -1630,6 +2244,6 @@
   },
   "timezone": "",
   "title": "NetApp Detail: Cluster",
-  "uid": "uc8GCF9Mk",
```

Comment:
do we need to change uid?

Reply:
doesn't matter. It will be anyways be marked as empty when imported through cli.